layer1/chains/ethereum/contracts.go

```diff
@@ -303,3 +315,23 @@ func logAndEat(logger *logrus.Logger, err error) {
 		logger.Error(err)
 	}
 }
+
+// Get the current validators.
+func GetValidators(
+	eth layer1.Client,
+	contracts layer1.EthereumContracts,
+	logger *logrus.Logger,
+	ctx context.Context,
+) ([]common.Address, error) {
+	callOpts, err := eth.GetCallOpts(ctx, eth.GetDefaultAccount())
+	if err != nil {
+		return nil, err
+	}
+	validatorAddresses, err := contracts.ValidatorPool().GetValidatorsAddresses(callOpts)
+	if err != nil {
+		logger.Warnf("Could not call contract:%v", err)
+		return nil, err
+	}
+
+	return validatorAddresses, nil
```

Comment:
Return not covered by test? Is there a need for the return and if so why is it not tested?